api/src/test/java/org/pmiops/workbench/firecloud/FireCloudServiceImplTest.java

```diff
@@ -185,4 +187,28 @@ public void testCreateAllOfUsBillingProject() throws Exception {
     assertThat(request.isHighSecurityNetwork()).isTrue();
     assertThat(request.getServicePerimeter()).isEqualTo(servicePerimeter);
   }
+
+  @Test
+  public void testCreateAllOfUsBillingProject_v2BillingApi() throws Exception {
+    final String servicePerimeter = "a-cloud-with-a-fence-around-it";
+    // confirm that this value is no longer how we choose perimeters
```

Comment:
this comment is no longer accurate - probably appeared after a rebase on a recent change I made